src/PrestaShopBundle/Bridge/Smarty/ToolbarFlagsConfigurator.php

```diff
@@ -56,15 +51,22 @@ class ToolbarFlagsConfigurator implements ConfiguratorInterface
     private $documentation;
 
     /**
-     * @param RouterInterface $router
+     * @var EnvironmentInterface
+     */
+    private $environment;
+
+    /**
      * @param Configuration $configuration
      * @param Documentation $documentation
      */
-    public function __construct(RouterInterface $router, Configuration $configuration, Documentation $documentation)
-    {
-        $this->router = $router;
+    public function __construct(
+        Configuration $configuration,
+        Documentation $documentation,
+        EnvironmentInterface $environment
```

Comment:
Why require environment when you only need isDebug()?
```suggestion
        bool $isDebugMode
```